cwms-data-api/src/main/java/cwms/cda/api/LevelsController.java

```diff
@@ -287,8 +282,13 @@ public void getAll(@NotNull Context ctx) {
             String timezone = ctx.queryParamAsClass(TIMEZONE, String.class)
                     .getOrDefault("UTC");
 
-            if ("2".equals(version)) {
+            String format = ctx.queryParamAsClass(FORMAT, String.class).getOrDefault("");
 
+            if (format.isEmpty())
```

Comment:
I'm not sure if we decided what should happen if there isn't a format parameter but the accept-header is for application/json;version=1.

Reply:
its possible that format.isEmpty check isn't enough

Reply:
you can fix that here or in a follow-up

Reply:
If the except header is `applciation/json;version=1` AND it's one of the endpoints that would've gone to the database or been formatted that way the system should treat it like `format=json` was provided.